XamarinCommunityToolkitSample/Pages/Views/MediaElementPage.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Seems this file should be inside src folder with other sample files.
Also, as I know, we already have MediaElementPage class

Reply:
Rebased